CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ listed in the changelog.
 
 ## [Unreleased]
 
+### Fixed
+
+- make create-kind-with-registry failed locally on mac ([#679](https://github.com/opendevstack/ods-pipeline/issues/679))
```

Comment:
FYI I added the `skip-changelog` label earlier as the changelog is meant as a end-user facing changelog. But now that you added it, also fine ;)